### PR TITLE
Adjust IOS CI job

### PR DESCRIPTION
**Pull request by jakub-gonet**
Add caching, enable IOS build on all branches

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -41,11 +41,9 @@ matrix:
     xcode_destination: platform=iOS Simulator,OS=12.2,name=iPhone X
     env: NODE_VERSION=10.15.3
 
-    branches:
-      only:
-      - master
```
Comment (osdnk) [inline .travis.yml]: The idea under this line was to make PR build only if PR is targeted to master. IMO it's a reasonable approach not to build every commit pushed to any branch in the repo.

**Comment by Szymon20000** [comment]
We don't have Podfile in the root directory so probably this won't work. There is also a problem with the Development Pods.  However, we always do "pod install" so they shouldn't get out of sync with the cached version(development pods are linked but new files are not if we don't call "pod install").  Maybe we should cache Pods like [here](https://docs.travis-ci.com/user/caching/#arbitrary-directories)?

File: .travis.yml (hunk 1 above)
Comment (jakub-gonet) [reply .travis.yml]: Fixed.

**Comment by osdnk** [comment]
What's the status?

**Comment by osdnk** [comment]
Basically looks good. Have you actually made a benchmark and observed a boost? If yes, can you share your results?

Please, fix conflicts

**Comment by jakub-gonet** [comment]
After fixing CI this PR is no longer relevant, I'm closing it.

**Pull request closed without merging**